docs/user/tutorial/60-admin.rst

```diff
@@ -44,12 +40,33 @@ Add an image to the Filer
 
 .. todo:: write up steps properly
 
+.. image:: images/new_folder_button.png
```

Comment:
Please use Screenshot Node (in Firefox) or similar in Chrome so you don't get whitespace around buttons. Can you retake this image please?